src/gsi/gfs_stratosphere.f90

```diff
@@ -1153,11 +1155,17 @@ subroutine add_gfs_stratosphere
          ncdim = get_dim(atmges, 'grid_xt'); lonb = ncdim%len
          ncdim = get_dim(atmges, 'grid_yt'); latb = ncdim%len
          ncdim = get_dim(atmges, 'pfull'); levs = ncdim%len
+         write(6,*) 'ncio lonb latb lev =',lonb, latb, levs
```

Comment:
this print should probably again be inside an if statement checking PE

Reply:
Removed.